lib/commentOnClosedIssue.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+var Cesium = require('cesium');
+var Promise = require('bluebird');
+var rp = require('request-promise');
+
+var getUniqueMatch = require('./getUniqueMatch');
+
+var defined = Cesium.defined;
+
+/** Get comments -> regex search -> post comment
+ *
+ * @param {String} jsonResponse GitHub Response
+ * @param {Object} headers Request headers
+ * @return {Promise<http.IncomingMessage>} Response
+ */
+var commentOnClosedIssue = function(jsonResponse, headers) {
+    if (!defined(jsonResponse) || !defined(headers)) {
+        return Promise.reject('jsonResponse is undefined');
+    }
+    var commentsUrl = jsonResponse.issue.comments_url;
+    var comments = [];
+    var linkMatches = [];
+
+    return rp.get({
+        uri: commentsUrl,
+        headers: headers,
+        json: true
+    }).then(function(commentsJsonResponse) {
+        comments = commentOnClosedIssue._getCommentsFromResponse(commentsJsonResponse);
+        linkMatches = getUniqueMatch(comments, commentOnClosedIssue._googleLinkRegex);
+        if (linkMatches === []) {
+            return Promise.reject('No google group links found in comments!');
```

Comment:
That being said, using `console.log` to spit out a status message with relevant information is probably a good idea.